src/KanbanModal.js

```diff
@@ -34,22 +34,24 @@ const KanbanModal = (props) => {
             props.addTask(newTask);
           }}
         >
-          <div className="KanbanModal-input-container">
+          <div style={{display: 'flex'}}>
```

Comment:
We have a CSS file for each js file, why getting rid of the className...?

Reply:
the original css sucks, this is janky but gets the job done ;)

Reply:
I fail to see how having the display under the class is different from having it on the component. At worst you can insert an !important

Reply:
I can move all css changes to css files